lib/meson.build

```diff
@@ -10,7 +10,9 @@ else
 endif
 
 if cfg.get('HAVE_ICONV')
+   libiconv = cc.find_library('iconv', required: true)
    libfuse_sources += [ 'modules/iconv.c' ]
+   deps += [ libiconv ]
```

Comment:
This won't do anything, since it's overriden below

Reply:
I am sorry,it is my fault,In my work,i had move the define of deps up, but I forget commit it.